src/VMware/test/Remove-AzVMwarePrivateCloud.Tests.ps1

```diff
@@ -12,11 +12,16 @@ while(-not $mockingPath) {
 . ($mockingPath | Select-Object -First 1).FullName
 
 Describe 'Remove-AzVMwarePrivateCloud' {
-    It 'Delete' -skip {
-        { throw [System.NotImplementedException] } | Should -Not -Throw
+    It 'Delete' {
```

Comment:
where was this private cloud been created?

Reply:
We need to create two private Clouds to test the delete command separately.
One created in Get-AzVMwarePrivateCloud.
One created in New-AzVMwarePrivateCloud.

Reply:
we do not want to create and delete resources in separate test cases, we do not promise the order of test cases executed.

what it this test case executed before get and new?